File: AlgorithmJobs/5_1.시간복잡도/5-1k번째 큰 수 찾기_solution.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);

		int n = s.nextInt();
		int arr[] = new int[n];
		int k = s.nextInt();
		for (int i = 0; i < n; i++) {
			arr[i] = s.nextInt();
		}
		Arrays.sort(arr);
		System.out.print(arr[n - k]);

	
	}
}

File: AlgorithmJobs/5_1.시간복잡도/5-1k번째 큰 수 찾기_solution.cpp
#include <stdio.h>
#include <algorithm>
using namespace std;
int arr[100010];
int main(){
	int N,k;
	scanf("%d %d",&N,&k);
	for(int i=0;i<N;i++){
		scanf("%d",&arr[i]);	
	}
	sort(arr,arr+N);
	printf("%d",arr[N-k]);
}